exercises/01_penguin_classification.ipynb

```diff
@@ -57,7 +57,10 @@
     "\n",
     "The penguin data reading and processing can be encapsulated in a PyTorch dataset class.\n",
     "\n",
-    "- Why is a class representation helpful?\n",
+    "\n",
+    "<details>\n",
+    "<summary>Why is a class representation helpful?</summary>\n",
+    "\n",
```

Comment:
Some of the `...` were spaces to be completed by demonstrator/participants as they worked through the notebook. Are you still planning on doing this as part of the workshop, as if not these perhaps need completing (see worked solution for answers)?

Reply:
Yes, I agree. I left this question as a discussion point, but could add an ellipsis.